components/consent/consent.css

```diff
@@ -0,0 +1,71 @@
+@keyframes slideUp {
+  0% {
+    transform: translateY(100%);
+    opacity: 0.5;
+  }
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+amp-consent {
+  background: var(--color-white);
+  box-shadow: rgba(0, 0, 0, 0.26) 0 0 19px 0;
+  animation: .5s ease-out 0s 1 slideUp;
+}
+
+amp-consent .ampstart-btn {
+  width: 9.5em;
+}
+
+.ampstart-consent-popup p a {
+  color: #b60845;
+}
+
+/* phones */
+@media (--breakpoint-xs) {
+  amp-consent a.ampstart-consent-close-btn {
+    font-size: 22px;
+    top: 15px;
+    right: 13px;
+  }
+  
+  .ampstart-consent-popup div {
+    flex-flow: column wrap;
+    align-items: center;
+    margin: 12px 25px 15px 12px;
+  }
+
+  .ampstart-consent-popup p {
+    padding: 5px 20px 5px 10px;
+    margin: 0 0 10px 0;
+  }
+}
+
+/* tablets and desktop */
+@media (--breakpoint-sm) {
+ .ampstart-consent-popup a.ampstart-consent-close-btn {
+    font-size: 25px;
+    left: 30px;
+  }
+
+  .ampstart-consent-popup div {
+    flex-flow: row;
+    height: 100px;
+    margin: 0 40px;
+  }
+
+  .ampstart-consent-popup p {
+    max-width: 41.25em;
+    padding: 0 25px 0 40px;
+    margin: 0;
+  }
+}
+
+@media (max-width: 320px) {
+  .ampstart-consent-popup p {
+    line-height: 1em;
+    font-size: .75em;
```

Comment:
both this should be taken care by the typography classes?

Reply:
I wasn't sure how to do that within a media query...

Reply:
yeah there are no responsive typography classes - may be discuss with abby on this  (non blocking) but use the --h* variables (look in the typography.css) instead of the actual values. 

FYI: we dont use ems - we use rems.